### DISCUSSION TITLE
Support delete and insert data operations?

### DISCUSSION THREAD

**Discussion by xaljer** [jovibor/HexCtrl#77] [Ideas]
delete and insert are basic and useful operations.

**Comment by jovibor** [comment]
I think you misunderstand the point of the HexCtrl. It can not insert or delete anything (besides standard copy/paste), it's absolutely data agnostic.
It's just a control that you can build your app on top of. You can think of it as a very advanced `edit box`.
The sample app (`.exe`) is provided just for the demonstration purposes.

**Comment by xaljer** [comment]
I see, thanks for your reply.
